### ISSUE TITLE
Flutter's website has old/incorrect code in widgets-intro.md

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter#6685]
Error:

```
18 code snippets extracted.
Analyzing the extracted Dart libraries.
Analyzing 18 files...
[warning] Instance member 'white' can't be accessed using static access. (example/widgets_intro_md_2.dart, line 43, col 64)
(Ran in 11.2s)
```

**Comment by Hixie** [comment]
@cbracken This is yours.

**Comment by sethladd** [comment]
Should we check the website snippets when a PR is submitted to the main Flutter SDK?

**Comment by cbracken** [comment]
Fixed in flutter/website#316

**Comment by sethladd** [comment]
Thanks!
